services/pdf-ms/lambdas/addPdfToCase.ts

```diff
@@ -35,7 +35,7 @@ export const main = async (event: Record<string, any>) => {
       .filter(c => c.SK !== SK)
       .sort((c1, c2) => c2.updatedAt - c1.updatedAt);
```

Comment:
SK, c, c1, c2?!
pronounced names please

Reply:
Isn't the letter c perfectly pronouncable though? SK might not be pronouncable, but hey, we've used that as a property name all over the API, so it's not exactly originating here...

Also in my view, for these kind of short inline expressions, it's perfectly readable to have short variable names, since they are only there for this very short expression and not remembered past that.